core/src/autogluon/core/scheduler/seq_scheduler.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import time
+from collections import OrderedDict
+from copy import deepcopy
+
+from autogluon.core.searcher import GPFIFOSearcher
+from autogluon.core.utils import EasyDict
+from tqdm.auto import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+class LocalReporter:
+    def __init__(self, trial, config, training_history: dict, config_history: dict):
```

Comment:
I got really confused about config, can you call this something else? It is certainly not the config for which train_fn is called.

Reply:
this config retrieved from searcher:
```
searcher_config = self.searcher.get_config()
```
It goes both: here and in `train_fn` call:
```
reporter = LocalReporter(task_id, searcher_config, self.training_history, self.config_history)
...
self.train_fn(args, config=searcher_config, reporter=reporter)
```
Renamed to `searcher_config`